l10n_es_aeat_mod190/tests/test_mod190.py

```diff
@@ -32,6 +32,15 @@ def setUpClass(cls):
                 "a_nacimiento": "2000",
             }
         )
+        # cls.supplier.write(
```

Comment:
Why this commented code?

Reply:
Hello @pedrobaeza 
Because i'am check for another test case

Reply:
You should respect existing test cases, and maybe add new ones. And never leave commented code. Just remove it, but as said, not for this case.